server/plugin/command.go

```diff
@@ -314,7 +325,30 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 			return err.Error()
 		}
 
-		return fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		var subOrgMsg = fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		if flags.ExcludeOrgRepos {
+			var excludeMsg string
+			for _, value := range strings.Split(excludeRepo, ",") {
+				val := strings.TrimSpace(value)
+				notificationOffRepoOwner, NotificationOffRepo := parseOwnerAndRepo(val, p.getBaseURL())
+				if notificationOffRepoOwner != owner {
+					return fmt.Sprintf("--exclude repository  %s is not of subscribed organization .", NotificationOffRepo)
+				}
+				if err := p.StoreExcludedNotificationRepo(val); err != nil {
+					return err.Error()
+				}
+				if excludeMsg != "" {
+					excludeMsg += " and " + NotificationOffRepo
+					continue
+				}
+				excludeMsg += NotificationOffRepo
+			}
+			subOrgMsg += "\n\n" + fmt.Sprintf("But Notifications are disabled for %s", excludeMsg)
```

Comment:
I think the messaging is good, but would remove the `But`.![image](https://user-images.githubusercontent.com/7575921/128398065-fd7ce23e-5c29-47ae-8650-eb5a359d976a.png)
```suggestion
			subOrgMsg += "\n\n" + fmt.Sprintf("Notifications are disabled for %s", excludeMsg)
```

Reply:
This is outside the scope of this ticket, but we could hyperlink the organization and repos :)

Reply:
Sure , have added this too :)

Reply:
Awesome! Thanks!